### ISSUE TITLE
Current trunk 3ab34a0 doesn't build due to gusb

### ISSUE THREAD

**Issue by Mounium** [NixOS/nixpkgs#36741]
## Issue description
Trying to nixos-rebuild on current trunk 3ab34a0 or 14464b1 results in the following error: https://gist.github.com/Mounium/e699d6205bb469d66532c5229c102123  , it does build on hydra so I don't know what part of my configuration.nix could be the problem (also, why does it start to build it in the first place), there is nothing unusual in it, I tried disabling some settings that may cause the problem like nixUnstable package, but they don't solve the problem.

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.15.7, NixOS, 18.09.git.bea585c (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0`
 - nixpkgs: 3ab34a0 or 14464b1

**Comment by pbogdan** [comment]
Fails on Hydra too as seen in https://hydra.nixos.org/build/70806569/nixlog/1 (and for me locally as well). Looks like something to do with recent gtk-doc update in eb0368554661385ae470e245e50bd0bc10693bf0 Got it building with:

```diff
diff --git a/pkgs/development/tools/documentation/gtk-doc/default.nix b/pkgs/development/tools/documentation/gtk-doc/default.nix
index 6e7b45b97f6..e8363f8425a 100644
--- a/pkgs/development/tools/documentation/gtk-doc/default.nix
+++ b/pkgs/development/tools/documentation/gtk-doc/default.nix
@@ -18,8 +18,9 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook ];
   buildInputs =
-   [ pkgconfig perl python libxml2Python libxslt docbook_xml_dtd_43 docbook_xsl
-     gnome-doc-utils dblatex gettext which itstool
+   [ pkgconfig perl (python.withPackages(ps: [ ps.six ])) libxml2Python libxslt
+     docbook_xml_dtd_43 docbook_xsl gnome-doc-utils dblatex gettext which
+     itstool
    ];
 
   configureFlags = "--disable-scrollkeeper";
```

but as I don't know much about Python-related stuff I'm not sure if that's the right way of fixing this.

**Comment by pbogdan** [comment]
Ah, nvm, i now see there is a PR opened already - https://github.com/NixOS/nixpkgs/pull/36743.

**Comment by FRidh** [comment]
After a little mistake the fix is now in staging (4eec6bc9c9cafac55dbf774a2fd30da7e5a79cc4). Keeping open till it lands in master.

**Comment by xeji** [comment]
After a quick back-and-forth, the fix is in master as of https://github.com/NixOS/nixpkgs/commit/b3d2a87b53219589e0f3db57a3efe587eca9b299 . Just built gusb with it.
